src/10-Introduction.md

```diff
@@ -0,0 +1,21 @@
+## 1.0 Introduction
+
+### So what is this guide for ?
+It's for people to read how we working in PAiP Web and what rules we are trying restrict ourselves too for making better code.
+
+### Why i created this guide ?
+It's for making my CR easier process and making place rules for making better code which i find along my path of Developer.
+
+### What this guide will cover ?
+Firstly we will take a look at some Git Workflow and Git itself to make everything easy.
```

Comment:
We need TOC (Table of Contents) somewhere to do not only this way of explaining. But it will be to do near end probably.